### PR TITLE
.travis.yml: Update Golang version to 1.11.5

### PR DESCRIPTION
**Pull request by frextrite**
Removed old golang repository links from .travis.yml

Setup gimme to install the latest stable version of golang
v1.11.5 available at the time of the change

Closes https://github.com/coala/coala-bears/issues/2843

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `corobo mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

File: .ci/deps.go.sh [hunk 1]
```diff
@@ -2,8 +2,7 @@
 
 set -ex
 
-go get -u github.com/alecthomas/gometalinter
+curl -L https://git.io/vp6lP | sh
+mv bin/ ~
 
-gometalinter --install
-
-go get -u github.com/BurntSushi/toml/cmd/tomlv
```
Comment (Naveenaidu) [inline .ci/deps.go.sh]: Why is this line removed? Any specific reasons?

File: .ci/deps.go.sh [hunk 2]
```diff
@@ -2,8 +2,7 @@
 
 set -ex
 
-go get -u github.com/alecthomas/gometalinter
+curl -L https://git.io/vp6lP | sh
```
[the remark this answers is not in the document]
Comment (jayvdb) [reply .ci/deps.go.sh]: Did the old way work?  If so, keep your unbreaking change to only be the bit that unbreaks it.

What gometalinter encourages or discourages is irrlevant for our CI.  They encourage what is maintainable for them.  But we maintain our own CI, not them, and we dont beg them for help.

Also please rebase on top of https://github.com/coala/coala-bears/pull/2848 so we can see it working

**Comment by frextrite** [comment]
Rebasing on top of #2848 

Next step: Update golang and rollback gometalinter.

File: .ci/deps.go.sh (hunk 2 above)
[the remark this answers is not in the document]
Comment (jayvdb) [reply .ci/deps.go.sh]: What will eventually happen is interesting, and guides our CI structure a bit, but the purpose of CI is to break.

However, when fixing a breakage, the commit *must* only do the things absolutely necessary to fix the breakage.  Then the commit describes the fix, not the fix and some other nice stuff.  In the future, someone will be wanting to understand why you made the changes you made, and it will take much longer to understand if they can not see a green build containing only the minimal changes needed to unbreak it.

**Comment by frextrite** [comment]
@jayvdb some of the labels and assignee was accidentally changed by gitmate. Please fix them.

**Comment by jayvdb** [comment]
ack https://github.com/coala/coala-bears/pull/2850/commits/bf295680623dfef5b6fd88d9ea91493cd4e2b453

**Comment by jayvdb** [comment]
@gitmate-bot ff
